common/djangoapps/config_models/migrations/0001_add_config_models_service_user.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import migrations
+
+
+USERNAME = settings.CONFIG_MODEL_SERVICE_WORKER_USERNAME
```

Comment:
This is the same thing done [here](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/commerce/migrations/0001_data__add_ecommerce_service_user.py). Please advise if there is a better approach.